api/assets/v2/assets/assetresponse.proto

```diff
@@ -0,0 +1,134 @@
+// Maintainers, please refer to the style guide here:
+//     https://developers.google.com/protocol-buffers/docs/style
+syntax = "proto3";
+package archivist.v2;
+option go_package="github.com/rkvst/go-rkvstcommon-api-gen/assets/v2/assets;assets";
```

Comment:
Could we take the opportunity of calling this package 'assetsv1api' and get rid of all the renaming that occurs when importing from other packages?

Reply:
That would lead to imports being temporarily different in forestrie vs avid so I'm reluctant to do that until we have both avid and forestrie aligned with this new repo. Happy to do that as a follow up tho. 

I think the big lift here is factoring the monster proto into smaller lumps, and that is a good start ?

(Give a mouse a cookie ;-)